hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -6,11 +6,7 @@ object HealthTransformations {
 
   /** Parse all health-condition-related fields out of a raw RedCap record. */
   def mapHealthConditions(rawRecord: RawRecord): Iterable[HlesHealthCondition] =
-    Iterable.concat(
-      mapInfectiousDisease(rawRecord),
-      mapEyeDisease(rawRecord),
-      mapCongenitalEyeDisorder(rawRecord)
-    )
+    healthConditions.flatMap(healthCondition => processHealthCondition(rawRecord, healthCondition))
```

Comment:
If you want:
```suggestion
    healthConditions.flatMap(processHealthCondition(rawRecord, _))
```
:scala: